src/Admin/Extensions/Registry.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Registry of WPGraphQL Extensions
+ *
+ * @todo This will eventually be replaced with a server registry.
+ *
+ * @package WPGraphQL\Admin\Extensions
+ * @since @todo
+ */
+
+namespace WPGraphQL\Admin\Extensions;
+
+/**
+ * Class Registry
+ *
+ * @phpstan-type Extension array{
+ *  name: non-empty-string,
+ *  description: non-empty-string,
+ *  plugin_url: non-empty-string,
+ *  support_url: non-empty-string,
+ *  documentation_url: non-empty-string,
+ *  repo_url?: string,
+ *  author: array{
+ *   name: non-empty-string,
+ *   homepage?: string,
+ *  },
+ * }
```

Comment:
This is all we need to validate submissions programmatically with PHPStan.

And the entire file can be dropped once we have a server payload of data with minimal changes to `Extensions.php`